Adaptors/Amqp/tests/QueueStorageTests.cs

```diff
@@ -0,0 +1,236 @@
+// This file is part of the ArmoniK project
+//
+// Copyright (C) ANEO, 2021-2022. All rights reserved.
+//   W. Kirschenmann   <wkirschenmann@aneo.fr>
+//   J. Gurhem         <jgurhem@aneo.fr>
+//   D. Dubuc          <ddubuc@aneo.fr>
+//   L. Ziane Khodja   <lzianekhodja@aneo.fr>
+//   F. Lemaitre       <flemaitre@aneo.fr>
+//   S. Djebbar        <sdjebbar@aneo.fr>
+//   J. Fonseca        <jfonseca@aneo.fr>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published
+// by the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY, without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using Amqp;
+
+using ArmoniK.Core.Common;
+using ArmoniK.Core.Common.Injection;
+using ArmoniK.Core.Common.Tests.Helpers;
+
+using Microsoft.Extensions.Logging.Abstractions;
+
+using Moq;
+
+using NUnit.Framework;
+
+using System;
+using System.Collections;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArmoniK.Core.Adapters.Amqp.Tests;
+
+[TestFixture]
+public class QueueStorageTests
+{
+  private Options.Amqp options_;
+  public Options.Amqp Options_ { get => options_; set => options_ = value; }
+
+  private static Options.Amqp CreateDefaultOptions()
+  {
+    return new Options.Amqp()
+    {
+      Host = "localhost",
+      User = "guest",
+      Password = "guest",
+      Port = 5672,
+      CaPath = "somePath",
+      Scheme = "someScheme",
+      CredentialsPath = "somePath",
+      MaxPriority = 5,
+      AllowHostMismatch = false
+    };
+  }
+
+  [SetUp]
+  public void SetDefaultOptions()
+  {
+    /* These options are only to feed the QueueStorage constructor
+     * and they do not play any role in the how the connection is created,
+     * the later is defined in the  SimpleAmqpClientHelper class */
+    Options_ = CreateDefaultOptions();
+  }
+
+  [Test]
+  public async Task SimpleBrokerTest()
```

Comment:
This test does not test the adapter but only the library. Is it on purpose?

Reply:
It was a play test, I'll remove it.